sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1097,14 +1097,16 @@ void setLogSerializer(LogSerializer logSerializer) {
      * @param thread    thread where crash occurred.
```

Comment:
Thanks for the PR.

To follow the convention, please add this before the `@param thread` in the javadoc.
```
@return UUID uncaught exception's UUID.
```